docs/source/noise/row_noise.rst

```diff
@@ -7,13 +7,66 @@ Row-based Noise
 Row-based noise operates on one row of data at a time, for example by omitting
 or duplicating entire rows.
 
+Types of row-based noise:
+
+.. contents::
+   :local:
+
+Do not respond
+--------------
+
+Sometimes people don't respond to a census or survey questionnaire and are
+unable to be reached by other means such as telephone or personal visit. For the
+Decennial Census and household surveys such as the ACS and CPS, people are found
+to respond at different rates depending on demographics such as age, sex, and
+race or ethnicity.
+
+For each demographic subgroup in pseudopeople, we assumed the nonresponse rate
+in the Decennial Census was equal to the *net* rate of undercount (ignoring
+duplication) estimated in [Census_PES]_. Net undercount rates for age/sex were
+combined additively with rates for race/ethnicity, and demographic subgroups
+with resulting net overcounts (negative nonresponse rates) were given a
+nonresponse rate of 0. We assumed nonresponse in the ACS was the same as in the
+Decennial Census, since these are conducted similarly. We assumed that CPS had
+the same nonresponse pattern as the Decennial Census, but with a constant 27.6%
+added to the nonresponse rates, since that was the nonresponse rate for CPS in
```

Comment:
we just added 27.6 to all rates?  i would have expected something more complicated.

Reply:
Yeah, it seems like maybe we should instead scale all the rates up by an appropriate factor to preserve the relative differences in rates instead of the absolute differences. Perhaps do something like this for a future enhancement? @zmbc any thoughts?

Reply:
Oh, I think Zeb said one way to think of this is that the differential response rates that we computed for the census are the "unexplained" effects due to demographics, and then for CPS, we're adding on a constant non-response rate that can be explained by the mode of the survey.

Reply:
I just added some wording to try to explain this.